core/templates/pages/topic-viewer-page/practice-tab/practice-tab.component.html

```diff
@@ -1,16 +1,18 @@
 <div class="oppia-practice-tab-container">
   <md-card class="oppia-page-card oppia-long-text">
     <h2 class="tab-title">
+      <!-- TODO(orthodoxparadox): Fix translate-->
```

Comment:
No, this is not how we should proceed. If it should be a TODO, it should be linked to a filed issue. Otherwise who will keep the track of these? It is an open source org, so the contributors come and go and it would difficult for us to fix TODO's assigned to a gone contributor.

Having said that, I don't think we need a TODO here. The translate functionality can be achieved by changing `<span translate="I18N_TOPIC_VIEWER_MASTER_SKILLS"></span>` to `<span [innerHTML]="'I18N_TOPIC_VIEWER_MASTER_SKILLS' | translate"></span>`. Please take a look at the changes done similarly in this file: https://github.com/oppia/oppia/blob/develop/core/templates/pages/about-page/about-page.component.html#L31. You can also refer the complete PR here: https://github.com/oppia/oppia/pull/9398/files#diff-6d7aa7010687a44d52ec715649dd86abR2.I would suggest you take a pass over this PR as well, it will give you info on how the translate pipe was added :D.

Reply:
There were problems with this method. @srijanreddy98 said he needed a couple of days for this

Reply:
Done for now